API_Guidelines.md

```diff
@@ -73,15 +73,25 @@ Payload
 **Need documentation**. In the mean time [Finding Records](https://guides.emberjs.com/v2.10.0/models/finding-records/)
 
 ## POST
-`save()` will make a POST request *only* if the record is *new*.  
 
 ### Creating Records
 #### Request
 URL
 :   `apiHost.com/movies`  
 
 Ember Data Method
-:   `movie.save()`
+:  
+
+*`save()` will make a POST request only if the record is new.*
```

Comment:
what does this mean? where is the new record verified? if you're referring to the difference between set() and createRecord() i think you already covered it nicely below.